hotel-api-server/src/test/java/com/fponce/hotelapp/hotel/HotelControllerTest.java

```diff
@@ -72,4 +76,15 @@ public void createHotelWithInvalidParameters() throws Exception {
             ).andDo(print())
             .andExpect(status().isPreconditionFailed()).andReturn();
     }
+
+    //@Test
```

Comment:
Is it correct to have `@Test` annotation commented?

Reply:
No, my bad, I just fixed it. Thanks.